tests/UI/campaigns/functional/BO/08_design/06_imageSettings/04_sortAndPagination.js

```diff
@@ -0,0 +1,233 @@
+
+  creationTests.forEach((test, index) => {
+    describe(`Create image setting n°${index + 1} in BO`, async () => {
+      const createImageTypeData = new ImageTypeFaker({name: `todelete${index}`});
+
+      it('should go to add new image type page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToAddImageTypePage', baseContext);
+
+        await imageSettingsPage.goToNewImageTypePage(page);
+        const pageTitle = await addImageTypePage.getPageTitle(page);
+        await expect(pageTitle).to.contains(addImageTypePage.pageTitleCreate);
+      });
+
+      it('should create image type and check result', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'createImageType', baseContext);
+
+        const textResult = await addImageTypePage.createEditImageType(page, createImageTypeData);
+        await expect(textResult).to.contains(imageSettingsPage.successfulCreationMessage);
+
+        const numberOfImageTypesAfterCreation = await imageSettingsPage.getNumberOfElementInGrid(page);
+        await expect(numberOfImageTypesAfterCreation).to.be.equal(numberOfImageTypes + 1 + index);
+      });
+    });
+  });
+
+  // 2 : Pagination
+  describe('Pagination next and previous', async () => {
+    it('should change the item number to 20 per page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'changeItemNumberTo20', baseContext);
+
+      const paginationNumber = await imageSettingsPage.selectPaginationLimit(page, '20');
+      expect(paginationNumber).to.equal('1 2');
+    });
+
+    it('should click on next', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnNext', baseContext);
+
+      const paginationNumber = await imageSettingsPage.paginationNext(page);
+      expect(paginationNumber).to.equal('1 2');
+    });
+
+    it('should click on previous', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnPrevious', baseContext);
+
+      const paginationNumber = await imageSettingsPage.paginationPrevious(page);
+      expect(paginationNumber).to.equal('1 2');
+    });
+
+    it('should change the item number to 50 per page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'changeItemNumberTo50', baseContext);
+
+      const paginationNumber = await imageSettingsPage.selectPaginationLimit(page, '50');
+      expect(paginationNumber).to.equal('1');
+    });
+  });
+
+  // 3 : Sort image settings
+  describe('Sort image settings table', async () => {
+    const sortTests = [
+      {
+        args: {
+          testIdentifier: 'sortByIdDesc', sortBy: 'id_image_type', sortDirection: 'down', isFloat: true,
+        },
+      },
+      {args: {testIdentifier: 'sortByNameAsc', sortBy: 'name', sortDirection: 'up'}},
+      {args: {testIdentifier: 'sortByNameDesc', sortBy: 'name', sortDirection: 'down'}},
```

Comment:
```suggestion
      {
        args: {
          testIdentifier: 'sortByNameAsc', sortBy: 'name', sortDirection: 'up',
        },
      },
      {
        args: {
          testIdentifier: 'sortByNameDesc', sortBy: 'name', sortDirection: 'down',
        },
      },
```

Just to have the same thing that others.

Reply:
Done thx :+1: